tests/acceptance/conftest.py

```diff
@@ -99,6 +99,17 @@ def subgroup(gl: Gitlab, group: Group):
         gl.groups.delete(f"{group.full_path}/{subgroup_name}")
 
 
+@pytest.fixture(scope="class")
+def other_subgroup(gl: Gitlab, group: Group):
+    subgroup_name = get_random_name("subgroup")
```

Comment:
Let's add the following comment. Looking at the other fixtures above, it seems duplication is already known. Not sure what the fix is though.

```
# TODO: deduplicate this - it's a copy and paste from the above fixture
```

Reply:
Added.